extension/scripts/main.js

```diff
@@ -69,30 +102,58 @@ function loadWidget () {
 	close.dataset.trackingAction = 'close';
 	close.addEventListener('click', removeSelf, false);
 
-
 	holder.setAttribute('id', 'cpp-widget-holder');
 	document.body.appendChild(holder);
 
-	return {
+	update();
+
+	widget = {
 		close: removeSelf,
+		update
 	}
 }
 
-chrome.runtime.sendMessage({
-	method: 'isEnabled',
-	host: location.host
-}, response => {
-	if (response.enabled) loadWidget();
-});
+function beginTests () {
```

Comment:
Currently, all of the tests appear to be triggered by messages passing to and from various scripts? Do we need this function? Is it intended that we'll call this function to initialise all of the tests in the future?